tests/Integration/Resources/Catalog/Test_UserMetadata.py

```diff
@@ -0,0 +1,127 @@
+"""
+Test of multi-VO user metadata handling. Assumes a running Dirac instance with the (master?) FileCatalog
+"""
+
```

Comment:
BTW the spacing of this file is 4 instead of 2

Reply:
Fixed.